start.py

```diff
@@ -45,7 +70,7 @@ def set_up_nginx_files():
         'ADMIN_PORT', str(admin_port)
     ).replace(
         'ROOT', os.getcwd()
-    )
+    ).replace('XFRAMEOPTIONS', x_frame_options)
```

Comment:
For consistency, we might want to also put this `.replace` on a new line as the ones above do?